Readme.rst

```diff
@@ -1,13 +1,17 @@
-.. image:: https://travis-ci.org/getpelican/pelican-plugins.svg?branch=master
-    :target: https://travis-ci.org/getpelican/pelican-plugins
-
 Pelican Plugins
 ###############
 
-Beginning with version 3.0, Pelican supports plugins. Plugins are a way to add
-features to Pelican without having to directly modify the Pelican core. Starting
-with 3.2, all plugins (including the ones previously in the core) have been
-moved to this repository, so this is the central place for all plugins.
+**Important note:** We are in the process of migrating plugins from this monolithic repository to their own individual repositories under the new `Pelican Plugins`_ organization, a place for plugin authors to collaborate more broadly with Pelican maintainers and other members of the community. The intention is for all the plugins under the new organization to be in the new “namespace plugin” format, which means these plugins can easily be Pip-installed and recognized immediately by Pelican 4.5+ — without having to explicitly enable them.
+
+This transition process will take some time, so we appreciate your patience in the interim. If you would like to help speed up this transition, the following would be very helpful:
+
+* **If you find a plugin here that has not yet been migrated to the new organization**, create a new issue under this repository and communicate which plugin you would like to help migrate, after which a Pelican maintainer will guide you through the process.
```

Comment:
A rough up front guideline or a link to a model migrated repo would be nice and maybe result in less guidance.

Reply:
Good suggestions. I already have a mostly-written plugin migration guide, but it's not quite ready yet. I'll put it up as soon as it's finished, but I don't want that to hold up putting *something* at the top of this README. That said, I just added links to a migrated repo as well as the [Cookiecutter template](https://github.com/getpelican/cookiecutter-pelican-plugin).